components/org.wso2.carbon.user.mgt.workflow/src/main/java/org/wso2/carbon/user/mgt/workflow/userstore/UserStoreActionListener.java

```diff
@@ -86,8 +87,14 @@ public boolean doPreAddUser(String userName, Object credential, String[] roleLis
             String errorMessage = String
```

Comment:
Shall we move this `errorMessage` computation to the else block of the if condition in line #91, as this will be computed unnecessarily even when the log masking is enabled?